README.md

```diff
@@ -47,6 +59,8 @@ You can also provide an absolute path:
 eslint-find-new-rules ~/Developer/eslint-config-kentcdodds/index.js
 ```
 
+**Please note** that any tested ESLint config file must reside below your project's root.
```

Comment:
Configs that extend other configs (installed as a module) cannot be resolved successfully, when not within the same project.

Example:

```
some-path
|
+-- project-a
|     +-- package.json
|     +-- index.js
|     +-- node_modules
|            +-- eslint@1.2.3
|            +-- eslint-config-something-1
|
+-- project-b
      +-- package.json
      +-- index-2.js
      +-- node_modules
             +-- eslint@2.3.4
             +-- eslint-config-something-2
```

`eslint-find-new-rules` installed to `project-a` will successfully interpret `index.js` and `package.json` below its "project root" (some-path/project-a). It will also resolve `eslint-config-something-1` when being extended from, let's say `index.js`.
But when invoking `eslint-find-new-rules` (still installed to `project-a`) on config files from `project-b`, it will fail on resolving `eslint-config-something-2` when being referenced from `index-2.js`.
The ruleset of `eslint@1.2.3` may also differ from `eslint@2.3.4`, distorting the output of rules, that could be added.